src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs

```diff
@@ -93,36 +93,6 @@ public dynamic LoadDocument(object keyOrEnumerable, string collectionName)
                 keyOrEnumerable.GetType().FullName + ": " + keyOrEnumerable);
         }
 
-        protected IEnumerable<AbstractField> CreateField(string name, object value, bool stored = false, bool? analyzed = null)
```

Comment:
It's tricky because each static index inherits from this class, but since we are compiling them in runtime then you will not find any usages